src/App.vue

```diff
@@ -151,6 +151,7 @@ import { CompanyResources } from '@/resources'
 // Enums and Constants
 import { EntityTypes, FilingCodes, FilingStatus, RouteNames, NameRequestStates } from '@/enums'
 import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
```

Comment:
I would prefer to see this re-imported locally, like here:
https://github.com/bcgov/business-filings-ui/blob/1e77360722b40500baabe2e9e50b02b604ef864d/src/enums/index.ts#L16

Reply:
interesting! ok, will do 👍

Reply:
Updated in next commit

Reply:
I'm having a few issues doing this throughout. Specifically when using in other enums. Looking into this atm.

Reply:
Should be same as in Filings UI?